tests/integration/plugins/test_ant.py

```diff
@@ -0,0 +1,60 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright 2022 Canonical Ltd.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License version 3 as published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import subprocess
+import textwrap
+from pathlib import Path
+
+import yaml
+
+from craft_parts import LifecycleManager, Step
+
+
+def test_ant_plugin(new_dir, monkeypatch):
+    monkeypatch.setenv("CRAFT_PARTS_PACKAGE_REFRESH", "0")
+    source_location = Path(__file__).parent / "test_ant"
+
+    parts_yaml = textwrap.dedent(
+        f"""
+        parts:
+          foo:
+            plugin: ant
+            source: {source_location}
+            stage-packages: [default-jre-headless]
```

Comment:
I was surprised to see that this test passed on CI without me having to add `ant` and the jdk on the part.

Reply:
If jdk is pre-installed in the environment, can something unexpected happen if we use a different version at runtime? If so it could be interesting to make sure the stage package has a consistent version.